ODT/ODTUtility.php

```diff
@@ -194,20 +194,26 @@ public static function deleteUselessElements(&$docContent, array $preventDeletet
      *                Just the integer value, no units included.
      */
     public static function getImageSize($src, $maxwidth=NULL, $maxheight=NULL){
-        if(file_exists($src))
+        if(!file_exists($src))
         {
+            $fetch = (new DokuHTTPClient())->get($src);
+            if(!$fetch) {
+                return array(0, 0);
+            }
+        }
+
+        if(file_exists($src)) {
```

Comment:
Not really needed, I'd use those checks to make the code more easily to understand and more maintenable for other developers. I'd merge the first If, the other ones not sure if it would be possible without breaking the code as those checks are needed.